src/app/pages/app-bar/page-wide-search/page-wide-search.component.html

```diff
@@ -0,0 +1,70 @@
+<mat-toolbar data-cy="pxb-toolbar" color="primary" class="main-header mat-elevation-z4 t" [class.hidden]="toggleSearch">
```

Comment:
what does this `t` class do?

Reply:
And it looks like this shadow has gone missing
![image](https://user-images.githubusercontent.com/8997218/134049477-96e1582a-8714-4fd5-b96f-453388f47a32.png)

Reply:
I've re-added the elevation here.